HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -373,6 +373,46 @@ public IActionResult ConfirmLocalAuthority_Post(ConfirmLocalAuthorityViewModel v
         return RedirectToNextStep(nextStep, viewModel.EntryPoint);
     }
 
+    [HttpGet("not-taking-part")]
+    public IActionResult NotTakingPart_Get(QuestionFlowStep? entryPoint, bool emailPreferenceSubmitted = false)
+    {
+        var questionnaire = questionnaireService.GetQuestionnaire();
+        var viewModel = new NotTakingPartViewModel()
+        {
+            LocalAuthorityName = questionnaire.LocalAuthorityName,
+            Submitted = emailPreferenceSubmitted,
+            EntryPoint = entryPoint,

```

Comment:
Please also set CanContactByEmailAboutFutureSchemes and Email address here from the questionnaire

Reply:
I've also made this change to the ineligible page :)

Reply:
Nice, thanks